### PR TITLE
Add twig-extensions extensions as services

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| License       | MIT

see symfony/symfony#22849

### CODE REVIEW

**Comment by javiereguiluz** [comment]
This looks a bit confusing to me. Twig extensions are a different project/repo from Twig, so if you uncomment this, you'll see an error (unless you install extensions). And having this config if you don't use extensions, looks strange.

**Comment by fabpot** [comment]
@javiereguiluz You're right. Perhaps we need to create a `twig_extensions.yml` file that is installed only when `twig/extensions` is installed. WDYT?

**Comment by fabpot** [comment]
@javiereguiluz Does it look better?

File: twig/extensions/1.0/config/packages/twig_extensions.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    #Twig\Extensions\ArrayExtension: ~
+    #Twig\Extensions\DateExtension: ~
+    #Twig\Extensions\I18nExtension: ~
```
Comment (stof) [inline twig/extensions/1.0/config/packages/twig_extensions.yaml]: this one should not be included IMO, as it conflicts with the Symfony TranslatorExtension

File: twig/extensions/1.0/config/packages/twig_extensions.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+services:
+    _defaults:
```
Comment (stof) [inline twig/extensions/1.0/config/packages/twig_extensions.yaml]: should also have `public: false`

Comment (fabpot) [reply twig/extensions/1.0/config/packages/twig_extensions.yaml]: fixed

File: twig/extensions/1.0/config/packages/twig_extensions.yaml (hunk 1 above)
Comment (fabpot) [reply twig/extensions/1.0/config/packages/twig_extensions.yaml]: removed

**Pull request merged**